test/integration/provider/admin/api_docs/account_data_controller_test.rb

```diff
@@ -6,6 +6,10 @@ class Provider::Admin::ApiDocs::AccountDataControllerTest < ActionDispatch::Inte
   def setup
     @provider = FactoryBot.create(:provider_account)
     @service = provider.default_service
+
+    provider.backend_apis.last.delete # FIXME: provider_account has 2 backend apis!
```

Comment:
![image](https://user-images.githubusercontent.com/11318903/78255167-4022fb80-74f7-11ea-92e3-4fffd3b99ef0.png)

Reply:
Fixed in https://github.com/3scale/porta/pull/1805 😄

Reply:
Let's get that one merged first and then I'll rebase this one and remove this line.